### DISCUSSION TITLE
如何修改目录Abstract字体和页码字号

### DISCUSSION THREAD

**Discussion by frelwx** [sjtug/SJTUThesis#972] [提问]
![image](https://github.com/sjtug/SJTUThesis/assets/57762884/60347ce8-e6d1-443b-9532-675657de4100)
上面为图书馆老师的意见。
意见2：我根据[#942](https://github.com/sjtug/SJTUThesis/discussions/942#discussion-5204225)在目录中添加了中英文摘要，但是字体不是Arial，请问该如何修改呢？
![image](https://github.com/sjtug/SJTUThesis/assets/57762884/cb3ec640-cc3b-4135-a2fc-9f8b1a86df08)
![image](https://github.com/sjtug/SJTUThesis/assets/57762884/7f5104eb-3a28-42ce-8780-b23cdc3d2af5)
意见3：我之前根据[#963](https://github.com/sjtug/SJTUThesis/discussions/963#discussion-5269497)修改了多级目录缩进和字号，但是后面的页码数字并没有跟着字体发生变化。
![image](https://github.com/sjtug/SJTUThesis/assets/57762884/806f5b61-d1d1-489a-8cd2-e4c23cdba5d7)

意见4：目录中的“参考文献”四字间距明显比一个空格大，请问如何修改。添加空格的方法是按照水源上帖子修改的
[标题“参考文献”加空格，页眉不加空格](https://shuiyuan.sjtu.edu.cn/t/topic/167561/16)
![image](https://github.com/sjtug/SJTUThesis/assets/57762884/a08d935b-bac2-4cd5-8ee5-bd5f201b48d9)

**Comment by LogCreative** [comment]
这不是精益求精，这是打算改的乱七八糟。建议跟审核人邮件交流。

**Reply by LogCreative** [reply]
沟通时请声明使用的是 LaTeX 模板，而不是 Word 模板，官方模板未实现的部分不需要实现。

一个正常的目录不应当有三种以上不同的字体，字号也不是必须要修正的。

**Reply by frelwx** [reply]
感谢，和专家邮件沟通后已通过，不需要修改。
